microcosm_flask/conventions/crud.py

```diff
@@ -49,9 +48,9 @@ def register_search_endpoint(graph, obj, path_prefix, func, request_schema, resp
     :param response_schema: a marshmallow schema to encode (a single) response item
     """
 
-    paginated_list_schema = make_paginated_list_schema(obj, response_schema)()
+    paginated_list_schema = make_paginated_list_schema(ns, response_schema)()
 
-    @graph.route(path_prefix + collection_path_for(obj), Operation.Search, obj)
+    @graph.route(ns.collection_path, Operation.Search, ns)
```

Comment:
All calls to the internal router now expect a `path`, an `Operation`, and ` Namespace`. In most cases, the `path` should be obtained from a `Namespace` property.